microsoft/botframework-solutions
# [To Do] Fix bugs of new To Do flow

Fix serveral bugs of new flow
## Description
Fix serveral bugs of new flow
26995 Todo: keep showing the first page when user say "previous page" while todo is in the first page
27044 Todo: Add retry mechanism for yes/no confirm prompt
27046 Todo: Should not generate outlook task folder and send email when user have already have it

## Related Issue
#667

## Testing Steps
1. say read more after the next page or show previous in the first page to verify 26995
2. say anything besides "Yes/No" in any confirmation dialog to verify 27044
3. Login with a new account to trigger to do skill and then re-login with new users to verify 27046

## Checklist
- [x] I have tested any new/updated dialogs using Speech in the emulator to ensure the [Speak]
